docs/components/static-render-entry.jsx

```diff
@@ -0,0 +1,33 @@
+import { match, RoutingContext } from "react-router";
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+
+import IndexTemplate from "./static-index";
+import { routes } from "../router";
+
+const Index = React.createFactory(IndexTemplate);
+const _renderIndex = (component) => `<!DOCTYPE html>
+  ${ReactDOMServer.renderToStaticMarkup(component)}`;
+
+/**
+ * Helper component that allows `static-site-generator-webpack-plugin` to render
+ * app as static HTML
+ *
+ * Output built to `/gh-pages/`
+ */
+
```

Comment:
This is missing 3bc397b which is necessary for google analytics to work in the static build.

Please review the diff here to make sure all the changes made it through the rebase of `docs-googleAnalytics` from earlier today: https://github.com/FormidableLabs/victory/compare/docs...docs-googleAnalytics-beforeRebase?expand=1